packages/stryker/src/Stryker.ts

```diff
@@ -68,14 +69,15 @@ export default class Stryker {
 
   private mutate(inputFiles: File[], initialTestRunResult: InitialTestRunResult) {
     const mutator = new MutatorFacade(this.config);
-    const mutants = mutator.mutate(inputFiles);
-    if (mutants.length) {
-      this.log.info(`${mutants.length} Mutant(s) generated`);
+    const allMutants = mutator.mutate(inputFiles);
+    const includedMutants = this.removeExcludedMutants(allMutants);
+    if (includedMutants.length) {
+      this.log.info(`${includedMutants.length} Mutant(s) generated`);
```

Comment:
Could we change this log message to include the number of excluded mutants? 

```ts
const numberExcluded = allMutants.length - includedMutants.length;
this.log.info(`${includedMutants.length} Mutant(s) generated${numberExcluded ? ' (${numberExcluded} mutant(s) excluded)' : '' }`);
```

Same could be added to the mutant-free world message.